circuit_maintenance_parser/__init__.py

```diff
@@ -38,19 +38,21 @@
     Zayo,
 )
 
-SUPPORTED_PROVIDER_NAMES = [provider.get_provider_type() for provider in SUPPORTED_PROVIDERS]
-SUPPORTED_ORGANIZER_EMAILS = [provider.get_default_organizer() for provider in SUPPORTED_PROVIDERS]
+SUPPORTED_PROVIDER_NAMES = [provider().get_provider_type() for provider in SUPPORTED_PROVIDERS]
```

Comment:
`SUPPORTED_PROVIDER_NAMES` seems a little bit confusing when the function that is populating the list is calling `get_provider_type`. Ultimately the function returns a class name but do we envisage type being extended in any way. If so,  `SUPPORTED_PROVIDER_TYPES` might make more sense.